rosdep/python.yaml

```diff
@@ -1002,6 +1002,21 @@ python-httplib2:
     saucy: [python-httplib2]
     trusty: [python-httplib2]
     trusty_python3: [python3-httplib2]
+python-hypothesis:
+  debian:
+    sid: [python-hypothesis]
+    stretch: [python-hypothesis]
+  fedora:
+    '24': [python-hypothesis]
+    '25': [python-hypothesis]
+    '26': [python-hypothesis]
```

Comment:
Since all keys on Fedora are the same you could also make a generic entry (rather than making it distro specific): fedora: [python-hypothesis]